public/scripts/views/forms/chart.js

```diff
@@ -36,14 +39,7 @@
           },
           options: {
             responsive: true,
-            title: {
-              display: false,
-              text: "Stats"
-            },
-            legend: {
-              display: false
-            },
-            tooltips: {
+            tooltip: {
```

Comment:
`tooltip` needs to be under `plugins` to work 👍🏻

Reply:
It was working as it is too, but I saw the documentation, updated with tooltip under plugins